src/dcm-taskdefs.ant.xml

```diff
@@ -809,8 +809,8 @@
 
       <forxpath inprop="files" xpath="/downloadFiles/file" propname="filename">
         <sequential>
-          <dirname property="dirpath" file="@{local}@{filename}"/>
-          <mkdir dir="${dirpath}"/>
+          <dirname property="dirpath@{filename}" file="@{local}@{filename}"/>
```

Comment:
Will this work?

I don't know the resolving logic here to be confident on how this will function.

Reply:
Yes this works. I have tested it and customer has also acknowledged it.

Reply:
Here in xml file, which contains filename tags has file paths , which needs to be downloaded.
dirname tag mentioned above need to be updated to point to subsequent path in xml. But as dirname tag does not allow to override the "dirpath" property value, it points to same file path on each iteration. So i changed it to create different property each time using  property="dirpath@{filename} and this worked.